Languages/C/reverse.c

```diff
@@ -0,0 +1,15 @@
+#include<stdio.h>  
+ int main()    
+{    
+int n, reverse=0, rem;    
+printf("Enter a number: ");    
+  scanf("%d", &n);    
+  while(n!=0)    
+  {    
+     rem=n%10;    
+     reverse=reverse*10+rem;    
+     n/=10;    
+  }    
+  printf("Reversed Number: %d",reverse);    
+return 0;  
+}
```

Comment:
we
//we can reverse an array as well using similar algo//         #include<stdio.h>
int main()
{
     int n, c, d, a[10],[10];
     printf("enter the no of elemets of array");
     scanf("%d",&a[c]);
    printf("\n enter elemets of array");
    for(c=0;c<;c++)
        scanf("%d",&n);
        for(c=n-1,d=0;c>0;c--;d++)
         b[d]==a[c];
    for(c=0;c<n;c++)
     a[c]=b[c];
   for(c=0;c<n;c++)
    printf("\n reversed array is: ")
    return 0;
}